### PR TITLE
Blog post: AWS Lambda & Vertx Framework URL Shortener Backend

### PR DESCRIPTION
**Pull request by pendula95**
Ready for review and publish.

### CODE REVIEW

File: src/site/blog/posts/2017-11-13-AWS-Lambda-and-Vertx-Framework-URL-Shortener-Backend.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: AWS Lambda & Vertx Framework URL Shortener Backend
```
Comment (vietj) [inline src/site/blog/posts/2017-11-13-AWS-Lambda-and-Vertx-Framework-URL-Shortener-Backend.md]: can you change to Eclipse Vert.x based Framework URL shortened Backend ?

Comment (pendula95) [reply src/site/blog/posts/2017-11-13-AWS-Lambda-and-Vertx-Framework-URL-Shortener-Backend.md]: Yes, no problem. Done.

**Comment by tsegismont** [comment]
@vietj anything left needed before publishing?

**Comment by vietj** [comment]
@tsegismont nope

**Comment by tsegismont** [comment]
@vietj ok, I'll publish it tomorrow then

**Comment by tsegismont** [comment]
@pendula95 Thank you! Your post is published http://vertx.io/blog/eclipse-vert-x-based-framework-url-shortener-backend/

**Comment by pendula95** [comment]
Thank you. One question. I saw the post live the other day, now I can not find it while browsing the Blog on vertx.io. I see that draft tag has been set to true. Is this intentionally?

**Comment by tsegismont** [comment]
Apologies. I did not push my local changes and they were overridden on the next deployment. The blog is fixed now.

**Pull request merged**